infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/LogReplicationConfig.java

```diff
@@ -30,6 +30,9 @@
     // Log Replication default max data message size is 64MB
     public static final int MAX_DATA_MSG_SIZE_SUPPORTED = (64 << 20);
 
+    // Log Replication default max cache size (number of entries)
+    public static final int MAX_CACHE_SIZE_ENTRIES = 200;
```

Comment:
nit - adding both size and entries in the name is a bit confusing whether this refers to size of cache(200mb) or num entries.  What do you think?

Reply:
done!